tensorflow/python/data/experimental/benchmarks/map_and_batch_benchmark.py

```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Benchmarks for `tf.data.experimental.map_and_batch()`."""
+"""Benchmarks for `tf.data.experimental.map_and_batch()`.
+
+NOTE: Eager support is not added to these benchmarks as the `map_and_batch`
```

Comment:
I would be in favor of adding the eager support the functionality is not deprecated. What is deprecated is the way to invoke it through the public API (using `tf.data.Options` to specify which optimizations to run instead of applying the fusion manually).

Reply:
sure, I have refactored it.